packages/store/src/symbols.ts

```diff
@@ -144,6 +146,14 @@ export interface NgxsOnInit {
   ngxsOnInit(ctx?: StateContext<any>): void | any;
 }
 
+/**
+ * On change interface
+ */
+export interface NgxsOnChanges {
+  isFirstChange?: boolean;
```

Comment:
What do you think? Should we make this match the angular interface by calling this prop `firstChange`?
https://angular.io/api/core/SimpleChange

Reply:
I don't see the need for the function that they have on their interface other than encapsulation.

Reply:
isFirstChange() - method not needed

Reply:
yeah, but we have named our property(`isFirstChange`) after their method(`isFirstChange`), not after the Angular property(`firstChange`).

Reply:
We can name this property differently, it is necessary to track changes.

Reply:
Yeah, I was just wondering if it was the intention to make it as close to the Angular interface as possible?